pennylane/__init__.py

```diff
@@ -24,6 +24,16 @@
 import pennylane.compiler
 from pennylane.compiler import qjit
 import pennylane.capture
+import pennylane.decomposition
```

Comment:
Also, should we move this below `import pennylane.operation` in the import order since it depends on it?

This will be a bigger task for next quarter to flesh out the actual depedency tree, but I do think we should start ordering this while by what modules need to get imported first.

Reply:
I wasn't aware that the order matters here, I'll update.

Reply:
Done

Reply:
Historically, it's been hidden behind doing `import pennylane as qml` everywhere. But once you try and start resolving our circular dependency issues, the order here starts to matter a lot.